mautic/mautic
# Truncate page hit title

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

If you use page with tilte longer like 191 chars, then tracking pixel is not triggered. This PR fixed it



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create page with title longer like 191 chars like this

```html
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <title>Title longer than 191 chars Title longer than 191 chars Title longer than 191 chars Title longer than 191 chars Title longer than 191 chars Title longer than 191 chars Title longer than 191 chars Title longer than 191 chars Title longer than 191 chars Title longer than 191 chars Title longer than 191 chars Title longer than 191 chars Title longer than 191 chars </title>

</head>
<body>

</html>
```
3. Put tracking pixel on the page
4. Open the page
5. Tracking pixel is not triggered
6. After PR tracking pixel is triggered with title of page truncated to 191 chars




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10709"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
